### ISSUE TITLE
PostNotifications do not take an array of player_ids on Android

### ISSUE THREAD

**Issue by camin-mccluskey** [OneSignal/react-native-onesignal#926]
**Description:**

On Android (but not iOS) you cannot pass an array of `player_id` strings to the `postNotification` method. Doing so raises a `com.facebook.react.bridge.readablenativemap cannot be cast to java.lang.string` error. You can pass individual `player_id` strings (one at a time) to the postNotification but this workaround is a little messy and means you need to treat the SDK differently for iOS and Android. It seems that the underlying method calls `JSON.stringify` on all parameters but the `player_id` parameter. Stringifying the array of `player_id`s before passing them to the method prevents the error but results in a `404` - ` HTTP code: 400 create notification failed {"errors":["Incorrect player_id format in include_player_ids (not a valid UUID)`. Does anyone have a suggested workaround for this issue that would result in the expected behaviour for both platforms?

**Environment**

1. `OneSignal React-Native SDK: v3.5.0`
2.  `React-Native: v0.61.5`

**Steps to Reproduce Issue:**

1. Install OneSignal SDK using npm
2. Intialize the SDK for react native
3. Send a `postNotification` using an array of `player_ids`

**Anything else:**

Relevant parts of the stacktrace given in the description but happy to supply more detail if needed.

**Comment by rgomezp** [comment]
Great catch @camin-mccluskey , thanks for your patience

**Comment by mikechoch** [comment]
Great catch, this will be fixed in release `3.6.3` and you should now expect both Android and iOS SDKs to want you to pass a JS array.

Example:
```
var playerIds = ["player_id_1", "player_id_2", "player_id_3", "player_id_4"];
OneSignal.postNotification(content, data, playerIds, other);
```
